tycho-core/src/main/java/org/eclipse/tycho/core/ee/ExecutionEnvironmentConfigurationImpl.java

```diff
@@ -24,7 +24,7 @@
 
 public class ExecutionEnvironmentConfigurationImpl implements ExecutionEnvironmentConfiguration {
     // Most likely best to always be the latest known supported EE
-    private static final String DEFAULT_EXECUTION_ENVIRONMENT = "JavaSE-11";
```

Comment:
I don't think its a good choice to change this to anything else as the minimum requirement for a tycho build.
Instead the default value should be derived from the current running build JVM.

Reply:
I just proposed this PR following the comment above it as J17 is currently the _latest known supported EE_ :)

Also, this PR is a quick-fix to the discussed issue #517, feel free to close it if it's not appropriate in any case :)